app/src/main/java/de/westnordost/streetcomplete/quests/building_type/AddBuildingType.java

```diff
@@ -0,0 +1,41 @@
+package de.westnordost.streetcomplete.quests.building_type;
+
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddBuildingType extends SimpleOverpassQuestType
+{
+	@Inject public AddBuildingType(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters() { return "ways, relations with building=yes and !man_made and !building:description"; }
```

Comment:
Maybe exclude also location=underground to avoid confusing people by (frequently mistagged) underground features?